File: 800/734A/734A.cpp
#include <iostream>
using namespace std;

int main(){
    int n;

    cin >> n;

    char ch;
    int A_nums = 0;
    int D_nums = 0;

    for(int i = 0; i < n; i++){
        cin >> ch;

        if(ch == 'A'){
            A_nums++;
        }
        else{
            D_nums++;
        }
    }

    if(A_nums > D_nums){
        cout << "Anton" << endl;
    }
    else if(A_nums < D_nums){
        cout << "Danik" << endl;
    }
    else{
        cout << "Friendship" << endl;
    }
}

File: 734A/734A.py
n = int(input())

s = input()

A_nums = 0
D_nums = 0

for c in s:
    if c == 'A':
        A_nums += 1
    else:
        D_nums += 1

if A_nums > D_nums:
    print("Anton")
elif A_nums < D_nums:
    print("Danik")
else:
    print("Friendship")